### PR TITLE
Ignore rescheduled cards in old stats page

### PR DESCRIPTION
**Pull request by user1823**
Fixes https://github.com/open-spaced-repetition/fsrs4anki-helper/issues/320

Rescheduled cards are not actual reviews. So, including them in Cards Studied Today is inappropriate.

### CODE REVIEW

**Comment by user1823** [comment]
I am not sure what's the issue with the last commit. Help would be appreciated.

File: pylib/anki/stats.py [hunk 1]
```diff
@@ -746,9 +746,9 @@ def _eases(self) -> Any:
                 "id > %d" % ((self.col.sched.day_cutoff - (days * 86400)) * 1000)
             )
         if lims:
-            lim = "where " + " and ".join(lims)
+            lim = "where type != {REVLOG_RESCHED} and " + " and ".join(lims)
```
Comment (Gustaf-C) [inline pylib/anki/stats.py:749]: These should be f-strings:
```suggestion
            lim = f"where type != {REVLOG_RESCHED} and " + " and ".join(lims)
```

File: pylib/anki/stats.py [hunk 2]
```diff
@@ -746,9 +746,9 @@ def _eases(self) -> Any:
...
         else:
-            lim = ""
+            lim = "where type != {REVLOG_RESCHED}"
```
Comment (Gustaf-C) [inline pylib/anki/stats.py:751]:
```suggestion
            lim = f"where type != {REVLOG_RESCHED}"
```

Comment (L-M-Sherlock) [inline pylib/anki/stats.py:751]: It should be f"where type != {REVLOG_RESCHED}"

File: pylib/anki/stats.py (hunk 1 above)
Comment (L-M-Sherlock) [inline pylib/anki/stats.py:749]: It should be f"where type != {REVLOG_RESCHED} and "

**Review by L-M-Sherlock — CHANGES_REQUESTED** [review]

**Comment by dae** [comment]
Thank you both.

**Pull request merged**